### ISSUE TITLE
Delete key deferred until mouse moved or other key pressed

### ISSUE THREAD

**Issue by diefans** [wez/wezterm#219]
## Describe the bug

Delete key is in certain conditions not responding.

## Environment (please complete the following information):

 - OS: archlinux
 - Version: wezterm 20200608-110940-3fb3a61-17-g9f4e40b5

## To Reproduce

1. typing keys accidentaly with spaces between, e.g. `djasdnlsa dnsad k nsd sad nsna kdsandja`
2. jump to line start either with mutliple `ctrl-left` or `ctrl-a`
3. press delete `\33[3~` multiple times
4. nothing happens
5. if I move the mouse, all deferred key strokes take action

## Configuration

Be sure to include the relevant section(s) of your `wezterm.lua` configuration file.

## Expected behavior
Delete key takes action instantaneously.

## Session Recording

```
(NOW=`date -u +%Y%m%d%H%M%S` \
&& strace wezterm start -- script -t$NOW.timing $NOW.typescript 2>&1 | rg -v EAGAIN > $NOW.wezterm-del-deferred.txt \
&& tar -cz --remove-files -f $NOW.tar.gz $NOW.timing $NOW.typescript)
```
I have two session recordings one with deferred delete and one which works instantly.
the `txt` files contain strace recordings to see the difference

[20200612200855.tar.gz](https://github.com/wez/wezterm/files/4773072/20200612200855.tar.gz)
[20200612200855.wezterm-del-deferred.txt](https://github.com/wez/wezterm/files/4773073/20200612200855.wezterm-del-deferred.txt)
[20200612200934.tar.gz](https://github.com/wez/wezterm/files/4773074/20200612200934.tar.gz)
[20200612200934.wezterm-del-ok.txt](https://github.com/wez/wezterm/files/4773075/20200612200934.wezterm-del-ok.txt)

**Comment by wez** [comment]
Thanks for the report!  I believe that this is fixed in 11f599b0

**Comment by diefans** [comment]
works now - thanks!
